newscoop/src/Newscoop/NewscoopBundle/Controller/CommentsController.php

```diff
@@ -36,7 +36,7 @@ public function indexAction(Request $request)
         $blogService = $this->get('blog');
         $user = $userService->getCurrentUser();
 
-        if ($blogService->isBlogger($user)) {
```

Comment:
Yeah, but it shouldn't be connected to bloggers anyway - we should use here correct permission. It should be clear how to give someone access to it (even if he is a blogger).